### ISSUE TITLE
MultiText/CheckList property type enhancements

### ISSUE THREAD

**Issue by tomy2105** [elsa-workflows/elsa-core#887]
I'd like MultiText property type to be able to select from number of predefined values, at the moment I think you can add any text.
And if possible, would also like to have text/value mapping in these predefined values as is possible in Dropdown/CheckList.

In fact, if it would be better to explain this way, I'd like CheckList (finite number of options each with text/value) which is displayed in a MultiText way (not as series of checkboxes but number of tags you can add or remove). Is anything like this possible?

P.S. I've tried setting `UIHint` for my property to `DynamicList` hoping for the best, but my property didn't render at all, so this is probably not implemented yet, or I did something wrong?

**Comment by sfmskywalker** [comment]
Yep, makes sense.

**Comment by tomy2105** [comment]
Is DynamicList implemented?
Was it supposed to be something along the lines I described?
Or we need completely new UIHint/type?

**Comment by sfmskywalker** [comment]
Not implemented. To be honest I don't recall what I intended to do with it, but if I had to guess it ws supposed to be exactly what multi-text is today.

I don't think we need a separate UIHint for what you described. To me it sounds like it would just need a couple of additional settings to configure the item source (like we do for check-list) and whether or not the user is allowed to create new items on the fly.
